decaying_aggregator:
  # in_topics, crop_distances, and subscriber_queue_sizes must have the same length. The i-th element of each list corresponds to the i-th pointcloud.
  in_topics: [ "/scan_matched_points2", "/cameras/frontleft_depth/points_throttle", "/cameras/frontright_depth/points_throttle", "/cameras/back_depth/points_throttle" ] # The topics of the pointclouds to aggregate
  crop_distances: [ -1.0, 1.5, 1.5, 1.5 ] # The distances to crop the pointclouds to (negative values mean no cropping)
  subscriber_queue_sizes: [ 30, 1, 1, 1 ] # The queue sizes of the subscribers
  out_topic: "/decaying_cloud_aggregated" # The topic where to publish the aggregated pointcloud
  target_frame: "odom" # The target frame in which to aggregate the pointclouds
  publish_frame: "lidar_laser_frame" # The frame to publish the aggregated pointcloud in
  max_queue_size: 50 # The maximum queue size of the aggregator
  voxel_filter_size: 0.1 # The voxel filter size to downsample the aggregated pointcloud to
